posthog/management/commands/backfill_distinct_id_overrides.py

```diff
@@ -51,20 +54,52 @@ def execute(self, dry_run: bool = False) -> None:
                 parameters,
             )
 
+            # XXX: The RETURNING set isn't really useful here, but this QuerySet
+            # needs to be iterated over to force execution, so we might as well
+            # return something...
+            updated_teams = list(
+                Team.objects.raw(
+                    """
+                    UPDATE posthog_team
+                    SET extra_settings = COALESCE(extra_settings, '{}'::jsonb) || jsonb_build_object('distinct_id_overrides_backfilled', true)
+                    WHERE id = %s
+                    RETURNING *
+                    """,
+                    [self.team_id],
+                )
+            )
```

Comment:
I had assumed there would be a more convenient way to update this via the ORM… not sure the "shortcut" was worth it at this point.